doc/man1/openssl-pkeyutl.pod.in

```diff
@@ -450,6 +450,10 @@ A deterministic result can also be obtained by specifying an explicit
 entropy value via the B<hextest-entropy>:I<value> parameter.
 Deterministic B<ML-DSA> signing should only be used in tests.
 
+The B<context-string>:I<string> option can be used to specify a context string
```

Comment:
Also possible is `hexcontext-string` which enables you to specify the string in hex form (so you could include binary ctrl values in the string)